### ISSUE TITLE
Résumé: Improvements

### ISSUE THREAD

**Issue by rye** [rye/old-krye.io#168]
Some improvements that still need to be made ~after #167~ ASAP are:

- [ ] **Make the job titles bigger.** These are the position titles in the main column, and they should be circa 2pt&ndash;4pt taller.
- [ ] **Split paid/volunteer experience into two sections, or discretize the various different things such as projects and actual positions into their own sections.** These are just semantic changes to make it more clear.
- [ ] **Add relevant coursework to the college section.** The previous changes to the side column which added the college section was a good start, but relevant coursework should be included.
- [ ] **Include live demos of work.** Preferably on [archive.is](https://archive.is) for posterity, but y'know how that works.
- [ ] ~**Adapt old job descriptions to new `li` format.** This will become simpler after #167 is completed.~

Thanks to @cg505 for the feedback in Slack.  His feedback was made based off of [Résumé v6.1.pdf](https://github.com/rye/krye.io/files/1020006/Resume.pdf).

**Comment by rye** [comment]
This is deferred.
